bindings/python/tests/conftest.py

```diff
@@ -50,7 +50,9 @@ def setup_config(service_name):
     for key in os.environ.keys():
         if key.lower().startswith(prefix):
             config[key[len(prefix) :].lower()] = os.environ.get(key)
-
+    disable_random_root = os.environ.get("OPENDAL_DISABLE_RANDOM_ROOT", "True")
```

Comment:
```suggestion
    disable_random_root = bool(os.environ.get("OPENDAL_DISABLE_RANDOM_ROOT", "true")
```

Reply:
`OPENDAL_DISABLE_RANDOM_ROOT`'s default value is `false` which means we always enable `random_root` for services.

Reply:
In python, `bool("True")` will return `True`, `bool("False")` will return `True` as well

Reply:
> In python, `bool("True")` will return `True`, `bool("False")` will return `True` as well

Yep, let's check if `os.environ.get("OPENDAL_DISABLE_RANDOM_ROOT") == "true"`